scene/3d/skeleton_3d.cpp

```diff
@@ -195,10 +195,6 @@ void Skeleton3D::_get_property_list(List<PropertyInfo> *p_list) const {
 			p_list->push_back(pi);
 		}
 	}
-
-	for (PropertyInfo &E : *p_list) {
-		_validate_property(E);
```

Comment:
This is needed to validate the properties added above, but the way it was written, it would validate any property (including properties that get validated automatically, so they will be validated twice).

This code needs to be rewritten.

Reply:
If it's irrelevant for bound properties then yes.